archive.php

```diff
@@ -53,7 +53,10 @@
 				 * @since 0.5.4
 				 */
 				$title = apply_filters( 'largo_archive_rounduplink_title', __( 'Saved Links' , 'largo' ) );

```

Comment:
Collapse these two `is_post_type_archive` sections into one that runs the filter ` 'largo_archive_' . $post_type . '_title' `, and add a similar filter for the default feed url.